.github/actions/build-frontend/action.yml

```diff
@@ -16,13 +13,16 @@ inputs:
   api-endpoint:
     description: The endpoint to connect the frontend to an api
     required: true
+  node-version:
+    description: Node.js Version
```

Comment:
creating this new var so we dont have to set a global var and so we can make this action more reusable for other situations later on possibly.

Reply:
Is this really something we want in an action vs as a config in the docker file? I can't see a place where we'd want to just bump node version

Reply:
I agree! we should just do that in docker for sure. However currently we are currently not containerizing our frontend in the live environments, we're using just blob storage which requires us to npm build in CICD and push the build files to the Blob. I think the direction you Arin and I would like to go will change this is the future for prod environments and we can finally dockerize it.